README.md

```diff
@@ -18,60 +18,7 @@ Developer-friendly incident response with brilliant Slack integration.
 
 ## Getting Started
 
-We prepared multiple environments: [production](https://grafana.com/docs/grafana-cloud/oncall/open-source/#production-environment), [developer](DEVELOPER.md) and hobby:
-
-1. Download [`docker-compose.yml`](docker-compose.yml):
-
-```bash
-curl -fsSL https://raw.githubusercontent.com/grafana/oncall/dev/docker-compose.yml -o docker-compose.yml
-```
-
-2. Set variables:
-
-```bash
-echo "DOMAIN=http://localhost:8080
-COMPOSE_PROFILES=with_grafana  # Remove this line if you want to use existing grafana
-SECRET_KEY=my_random_secret_must_be_more_than_32_characters_long" > .env
-```
-
-3. Launch services:
-
-```bash
-docker-compose up -d
-```
-
-4. Issue one-time invite token:
-
-```bash
-docker-compose run engine python manage.py issue_invite_for_the_frontend --override
-```
-
-**Note**: if you remove the plugin configuration and reconfigure it, you will need to generate a new one-time invite token for your new configuration.
-
-5. Go to [OnCall Plugin Configuration](http://localhost:3000/plugins/grafana-oncall-app), using log in credentials as defined above: `admin`/`admin` (or find OnCall plugin in configuration->plugins) and connect OnCall _plugin_ with OnCall _backend_:
-
-```
-Invite token: ^^^ from the previous step.
-OnCall backend URL: http://engine:8080
-Grafana Url: http://grafana:3000
-```
-
-6. Enjoy! Check our [OSS docs](https://grafana.com/docs/grafana-cloud/oncall/open-source/) if you want to set up Slack, Telegram, Twilio or SMS/calls through Grafana Cloud.
-
-## Update version
-
-To update your Grafana OnCall hobby environment:
-
-```shell
-# Update Docker image
-docker-compose pull engine
-
-# Re-deploy
-docker-compose up -d
-```
-
-After updating the engine, you'll also need to click the "Update" button on the [plugin version page](http://localhost:3000/plugins/grafana-oncall-app?page=version-history).
-See [Grafana docs](https://grafana.com/docs/grafana/latest/administration/plugin-management/#update-a-plugin) for more info on updating Grafana plugins.
```

Comment:
personal nitpick here.. I felt that the instructions under "Getting Started" conflicted/duplicated instructions in `DEVELOPER.md`. I would propose just having the below message to point users to `DEVELOPER.md` if their goal is to run the project locally.

Reply:
I disagree with this.  I think the landing page for the project should have the most basic setup to run OnCall without following another link.  Most of the projects I use tell me how to get started on their landing page, having to go to another page is just an opportunity to lose people.  If I want to just try things out, reading my options are production or local development tells me this project isn't for me.

Reply:
discussed w/ @mderynck. Kept the basic instructions on `README.md` and modified them w/ the new startup instructions